app/Http/Controllers/BookingController.php

```diff
@@ -70,15 +70,30 @@ public function create(Request $request)
 
     }
 
-    public function all()
+    public function all(Request $request)
     {
-        $hidden = ['guests', 'updated_at'];
+        $search = $request->input('room_id');
```

Comment:
2 things, search isn't a great name, you should be more specific. What if we want to add an actual search later?

And it doesn't look like you're using the validator on this, you should be using it to make sure the room_id actually exists